import-service/src/functions/importFileParser/handler.ts

```diff
@@ -7,12 +7,13 @@ import * as AWS from 'aws-sdk';
 import csv from 'csv-parser';
 
 const importFileParser = async (event: S3Event) => {
-  console.log('start process importProductsFile', JSON.stringify(event.Records));
+  console.log('start process importFileParser', JSON.stringify(event.Records));
 
   const BucketName = process.env.BUCKET_NAME;
-  const s3 = new AWS.S3({ region: 'eu-west-1' });
+  const s3 = new AWS.S3({ region: 'eu-west-3' });
```

Comment:
it is better to pass already initialized S3 instance as a dependency to the function